Tests/js/views/assets/ez-searchlistview-tests.js

```diff
@@ -0,0 +1,222 @@
+            };
+            this.view.render();
+            Assert.isTrue(
+                templateCalled,
+                "The template should have used to render the view"
+            );
+
+
+            Assert.areNotEqual(
+                "", container.getHTML(),
+                "View container should contain the result of the view"
+            );
+        },
+
+        _assertColumns: function (columns) {
+            Assert.isArray(columns, "columns should be an array");
+            Assert.areEqual(
+                columns.length, this.view.get('displayedProperties').length,
+                "There should be a column per displayed property"
+            );
+            columns.forEach(function (col) {
+                Assert.isTrue(
+                    this.view.get('displayedProperties').indexOf(col.identifier) !== -1,
+                    "The column identifier should be in the displayedProperties attribute"
+                );
+                Assert.areEqual(
+                    this.view.get('propertyNames')[col.identifier],
+                    col.name,
+                    "The name should come from the propertyNames attribute"
+                );
+            }, this);
+        },
+
+        "Test available variables in the template": function () {
+            var origTpl = this.view.template,
+                that = this;
+
+            this.view.template = function (variables) {
+                Y.Assert.isObject(variables, "The template should receive some variables");
+                Y.Assert.areEqual(4, Y.Object.keys(variables).length, "The template should receive 4 variables");
+                Y.Assert.areSame(
+                    that.searchResultCount, variables.searchResultCount,
+                    "searchResultCount should be available in the template"
+                );
+                Y.Assert.areSame(
+                    that.items.length, variables.displayCount,
+                    "The displayCount should available in the template"
+                );
+                Y.Assert.areSame(
+                    that.limit <= that.searchResultCount - that.items.length ? that.limit : that.searchResultCount - that.items.length,
+                    variables.remainingCount,
+                    "The remainingCount should available in the template"
+                );
+                that._assertColumns(variables.columns);
+                return origTpl.apply(this, arguments);
+            };
+            this.view.render();
+        },
+
+        "Should render items": function () {
+            var container = this.view.get('container'),
+                listItems;
+            
+            this.view.render();
+            listItems = container.all('.ez-searchlist-content div');
+
+            Assert.areSame(
+                this.view.get('items').length,
+                listItems.size(),
+                "List items should be added to the template"
+            );
```

Comment:
actually most of those tests should use the tests *skeleton* defined in ez-loadmorepagination-tests